consensus/types/src/eth_spec.rs

```diff
@@ -585,6 +612,8 @@ impl EthSpec for GnosisEthSpec {
     type BytesPerCell = U2048;
     type KzgCommitmentsInclusionProofDepth = U4;
     type ProposerLookaheadSlots = U32; // Derived from (MIN_SEED_LOOKAHEAD + 1) * SLOTS_PER_EPOCH
+    type PTCSize = U64; // todo: verify if needs to be U512 for some reason like in Mark's OG implementation
```

Comment:
just wanted to check with you on this. pretty sure all good now but wanted to flag it anyways (https://ethereum.github.io/consensus-specs/specs/_features/eip7732/beacon-chain/#misc_1)

Reply:
Ohhhh no these types are special. The type is actually its value, not the number of bits in the type. Look at the `typenum` crate in rust.

Reply:
ahh, I see! `typenum` lets you express that the type represents `512` in this case, nice